inference-engine/src/vpu/graph_transformer/src/stages/static_shape_nms.cpp

```diff
@@ -119,6 +149,21 @@ void FrontEnd::parseStaticShapeNMS(const Model& model, const ie::CNNLayerPtr& la
 
     auto stage = model->addNewStage<StaticShapeNMS>(layer->name, StageType::StaticShapeNMS, layer, usedInputs, DataVector{outIndices, outShape});
     stage->attrs().set<bool>("center_point_box", centerPointBox);
+
+    auto spatDim = inputs[0]->desc().dim(Dim::H);
+
+    const int ALIGN_VALUE = 64;
+    const int buffer_size0 = 2 * sizeof(int16_t) * 4 * spatDim;
+    const int buffer_size1 = 2 * sizeof(int16_t) * spatDim;
+    const int buffer_size2 = 2 * sizeof(int32_t) * spatDim;
+    const int buffer_size = buffer_size0 + buffer_size1 + buffer_size2 + 2 * ALIGN_VALUE;
+
+    const auto& env = CompileEnv::get();
+    const auto numSlices = env.resources.numSHAVEs;
+
+    const int buffer_size3 = 4 * sizeof(int32_t) * 256;
```

Comment:
Could you move this line near to other buffer_size definitions, please?

Reply:
I have put this variable separately from the other to decrease the code distance between its declaration and its using.